packages/core/src/singleton/Auth/utils/index.ts

```diff
@@ -27,13 +27,12 @@ export function assertTokenProviderConfig(
 		assertionValid = false;
 	} else {
 		assertionValid =
-			!!cognitoConfig.userPoolClientId && !!cognitoConfig.userPoolClientId;
+			!!cognitoConfig.userPoolClientId && !!cognitoConfig.userPoolId;
 	}
-
 	return asserts(assertionValid, {
-		name: 'AuthTokenConfigException',
-		message: 'Auth Token Provider not configured',
-		recoverySuggestion: 'Make sure to call Amplify.configure in your app',
+		name: 'UserPoolClientIdConfigException',
+		message: 'UserPoolClientId not present in Config.',
+		recoverySuggestion: 'Make sure to provide UserPoolClientId in the config.',
 	});
```

Comment:
```suggestion
return asserts(assertionValid, {
		name: 'UserPoolConfigException',
		message: 'UserPoolClientId or userPoolId are not present in Config.',
		recoverySuggestion: 'Make sure to provide UserPoolClientId and UserPoolId in the config.',
	});
```
Wondering if we should improve this message as it might be miss leading if customers are passing the `clientId` but not the `userpoolId`